doc/dev/developer_guide/index.rst

```diff
@@ -1594,6 +1594,150 @@ server list`` on the teuthology machine, but the target VM hostnames (e.g.
 ``target149202171058.teuthology``) are resolvable within the teuthology
 cluster.
 
+Running Integration Tests using Teuthology
+==========================================
+
+Getting binaries
+----------------
+To run integration tests using teuthology, you need to have Ceph binaries
+built for your branch. Follow these steps to initiate the build process -
+
+#. Push the branch to `ceph-ci`_ repository. This triggers the process of
+   building the binaries.
+
+#. To confirm that the build process has initiated, spot the branch name
+   at `Shaman`_. Little after the build process has initiated, the single
+   entry with your branch name would multiply, each new entry for a different
```

Comment:
each new entry for a different combination of distro and flavor.

Reply:
Fixed it. Thanks!